Airports/EGBK/SMR/Regions.txt

```diff
@@ -0,0 +1,212 @@
+; - background
+REGIONNAME Northampton Sywell
+30982 N052.18.40.003 W000.47.50.029
+N052.18.33.919 W000.47.43.307
+N052.18.19.242 W000.47.59.146
+N052.18.13.166 W000.48.01.572
+N052.18.08.429 W000.47.58.138
+N052.18.04.214 W000.47.55.259
+N052.17.56.956 W000.47.54.590
+N052.17.53.961 W000.47.47.611
+N052.17.56.254 W000.47.36.657
+N052.17.56.800 W000.47.26.767
+N052.18.00.691 W000.47.20.669
+N052.18.18.767 W000.47.00.324
+N052.18.27.557 W000.46.56.476
+N052.18.35.255 W000.47.07.838
+N052.18.41.167 W000.47.13.313
+N052.18.51.673 W000.47.26.290
+; - 03R/21L
+REGIONNAME Northampton Sywell
+1396490 N052.18.08.342 W000.47.35.640
```

Comment:
Same as above, find a colour that is defined and works well, or define this as a colour.

Please do this for every colour that you have as numbers.